### PR TITLE
chore: update links/content to match Devicon

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Update links/content to match Devicon.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Thomas Bui
+Copyright (c) 2022 Devicons
```
Comment (Snailedlt) [inline LICENSE]: Not sure if we are allowed to change this. We at the very least have to make sure with @Thomas-Boi I think

**Review by Snailedlt — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (Panquesito7) [reply LICENSE]: Right. I'll just keep the year changed to 2022. Sounds good?

Comment (Snailedlt) [reply LICENSE]: No, the year should be the year the copyright started, so that shouldn't be changed either

Comment (Panquesito7) [reply LICENSE]: Alright. And not even `2020-2021`? The latest changes by Thomas were on that year.

Comment (Snailedlt) [reply LICENSE]: A date range is not needed, see [this stackexchange thread](https://opensource.stackexchange.com/questions/1522/what-should-be-written-in-mit-license-year-full-name) for more info.

More info about changing licenses: https://opensource.stackexchange.com/questions/8853/can-i-change-the-copyright-holder-name-and-year-of-an-mit-license

Comment (Panquesito7) [reply LICENSE]: Great research, thanks! :)
We should only change if we have permission from the copyright holder, in this case, Thomas Bui. But still, there's no harm in keeping this as-is.

Comment (Thomas-Boi) [reply LICENSE]: @Snailedlt and @Panquesito7 feel free to change the copyright to Devicon. The only thing that I request is that the code stays MIT license (so others can use it) and that I can use it in other places without any restriction 😃

**Review by Snailedlt — APPROVED** [review]

**Comment by Panquesito7** [comment]
Will be merging as-is for now. We can discuss this in another PR. Thanks, folks. :)

**Pull request merged**